plugins/rollbar_dashboards/app/views/rollbar_dashboards/_project.html.erb

```diff
@@ -0,0 +1,19 @@
+<%
+  unless project.rollbar_dashboards_settings.empty?
+    @dashboard_data = project.rollbar_dashboards_settings.map do |dashboard|
+      items = Rails.cache.fetch("project_#{dashboard.cache_key}", expires_in: 10.minutes) do
+        RollbarDashboards::Client.new(dashboard).top_errors&.first(SamsonRollbarDashboards::Engine::NUM_ITEMS)
+      end
+      [dashboard, items]
+    end
+%>
```

Comment:
might be simpler to cache the html instead of caching the data and then passing it in
then it could fetch the data in the partial